db/dataGenerator.js

```diff
@@ -0,0 +1,85 @@
+const faker = require('faker');
+const db = require('./dbConnection.js');
+
+const genUserData = () => {
+  const userArray = [];
+  // Create 50 users for each restaurant
+  for (let y = 0; y < 50; y++) {
+    const obj = {
+      username: faker.name.findName(),
+      location: faker.address.city(),
+      profilePicture: faker.image.avatar(),
+      vipStatus: faker.random.boolean(),
+      numberOfReviews: Math.floor(Math.random() * 200) + 1,
+    };
+    userArray.push(obj);
+    // console.log(userArray);
+  }
+  return userArray;
+};
+
+const genReviewData = () => {
+  const numberOfReviews = 5;
+  const masterReviewArray = [];
+  for (let z = 1; z < 101; z++) {
+    const eachRestReviewData = [];
+    for (let x = 1; x < numberOfReviews; x++) {
+      const obj = {
+        overallRating: Math.floor(Math.random() * 5) + 1,
```

Comment:
You can also just go Math.floor(Math.random() * 6) to achieve the same thing

Reply:
Ohh, will do that next time. Thanks for the tip!